framework/src/modules/chain/blocks/verify.js

```diff
@@ -65,14 +57,8 @@ class BlocksVerify {
 	}
 
 	async verifyBlockForger(block) {
-		// Check if block was generated by the right active delagate. Otherwise, fork 3
-		// DATABASE: Read only to mem_accounts to extract active delegate list
-		try {
-			await this.dposModule.verifyBlockForger(block);
-		} catch (error) {
-			this.roundsModule.fork(block, 3);
```

Comment:
Do we do this somewhere else now?

Reply:
I don't think so. It will be implemented as part of fork choice rule? @shuse2

Reply:
we don't need to do this `fork` record anymore because it's different fork choice rule and we can read it from the logs

Reply:
We maintained a database table for forks encountered by a node to gauge the health of the network. Do we no longer need it anymore? If so, do we/did we remove the corresponding table as well?

Reply:
We definitely should log the fork choices clearly, but I don't think we need to save it to the database, and it was caused the problem, too.
Let's remove the corresponding table as well

Reply:
Will remove the table `forks_stat` and its usage as well ;)